tests/kani/Vectors/main.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Ord, PartialOrd)]
+pub struct GuestAddress(pub u64);
+
+#[derive(Debug)]
+pub struct GuestRegionMmap {
+    guest_base: GuestAddress,
+}
+
+// TODO: running this with --unwrap 2 causes CBMC to hang in propositional reduction.
```

Comment:
Is this still the case? Do we have an issue for that?

Reply:
Looks like I messed up the commit. I removed this comment from `fixme_main.rs`, which should've been deleted, but kept it in `main.rs`. Should be fixed now.